packages/tangle/scheduler.go

```diff
@@ -132,6 +131,15 @@ func (s *Scheduler) Setup() {
 		if scheduled {
 			return
 		}
+		s.tangle.Storage.Message(messageID).Consume(func(message *Message) {
+			if clock.Since(message.IssuingTime()) > time.Minute {
```

Comment:
Maybe better to use some parameter to define the time threshold?

Reply:
Sure thing!